components/PostCardList/PostCardList.stories.tsx

```diff
@@ -105,6 +105,6 @@ const post: Post = {
       ]
     }
   } as unknown) as GameSituation,
-  createdAt: new Date(2020, 2, 4, 12, 15, 43, 100),
-  lastUpdatedAt: new Date(2020, 2, 4, 12, 15, 43, 100)
+  createdAt: new Date(),
+  lastUpdatedAt: new Date()
```

Comment:
Confirmed this is intended changes. With a view to showing "X seconds ago" because it's the longest string this story shows